stable/prometheus/README.md

```diff
@@ -271,6 +318,17 @@ Parameter | Description | Default
 `serverFiles.rules` | Prometheus server rules configuration | `{}`
 `serverFiles.prometheus.yml` | Prometheus server scrape configuration | example configuration
 `networkPolicy.enabled` | Enable NetworkPolicy | `false` |
+`sidecar.image.repository` | prometheus sidecar containers image | `kiwigrid/k8s-sidecar`
```

Comment:
Please change the image to `quay.io/coreos/prometheus-config-reloader:v0.22.0`. The source repo is [here](https://github.com/coreos/prometheus-operator/tree/be5b8c5905a708c03dce74aef33a095e6460c08d/cmd/prometheus-config-reloader)

Reply:
I had a look on the image and if I understand correctly it expects a env variable with the config file and reloads on config change. The sidecar container does a bit more. It watches for the for config maps with a certain label if it finds it then it adds/upgrades or removes the config afterwards it reloads the config.  Similar thing could probably be also added in to the prometheus-operator image but I'm not very proficient with go-lang.

Reply:
@gianrubio ?